styles/components/shared/infowindow.scss

```diff
@@ -44,3 +44,58 @@
     }
   }
 }
+
+.c-profile-tooltip {
```

Comment:
@pjosh reusing what you did on https://github.com/Vizzuality/TRASE-frontend/pull/343